flocker/node/_deploy.py

```diff
@@ -476,6 +484,7 @@ def got_volumes(available_manifestations):
                 available_manifestations.values())
 
             return [NodeState(
+                # XXX uuid=self.uuid,
```

Comment:
Oops.  I guess this actually works because of the deprecated automatic hostname-to-uuid logic.  Gets used here, gets used in whatever test exercises this code, pow victory.